lib/set_member_subject_selector.rb

```diff
@@ -11,7 +11,7 @@ def initialize(workflow, user)
 
   def set_member_subjects
     to_classify = select_set_member_subjects_to_classify
-    unless to_classify.exists?
+    if !to_classify.exists? && user
```

Comment:
I only want to run this if there main selection is empty as a fallback to queue data.